jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -162,3 +192,47 @@ def _get_subset_definition(self, *args):
         subset_state = subset_group.subset_state
 
         self._unpack_nested_subset(subset_state)
+
+    def vue_update_subset(self, *args):
+        if not self.is_editable:  # no-op
+            return
+
+        subset_group = [s for s in self.app.data_collection.subset_groups if
+                        s.label == self.subset_selected][0]
+        subset_state = subset_group.subset_state
+
+        # Composite region cannot be edited, so just grab first element.
+        subset_type = self.subset_types[0]
+        subset_definition = self.subset_definitions[0]
+
+        try:
+            if subset_type == "Range":
+                sbst_obj = subset_state
+            else:
+                sbst_obj = subset_state.roi
+
+            for d_att in subset_definition:
+                setattr(sbst_obj, d_att["att"], d_att["value"])
+
+            self.session.edit_subset_mode._combine_data(subset_state, override_mode=ReplaceMode)
```

Comment:
this might warrant an in-line comment... what does `_combine_data` do?

Reply:
Indeed. Took me a good 10 mins to recall why I picked this. I will add comment.

Reply:
Done.